python/alibiexplainer/alibiexplainer/anchor_tabular.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable, List, Dict, Optional
+import kfserving
+import logging
+import os
+import joblib
+import alibi
+from alibiexplainer.explainer_method import ExplainerMethodImpl
+import numpy as np
+import pandas as pd
+
+logging.basicConfig(level=kfserving.server.KFSERVER_LOGLEVEL)
+
+
+class AnchorTabular(ExplainerMethodImpl):
+
+    def __init__(self, predict_fn: Callable):
+        self.predict_fn = predict_fn
+        self.cmap = None
+        self.anchors_tabular: Optional[alibi.explainers.AnchorTabular] = None
+
+    def prepare(self):
+        training_data_url = os.environ.get("TRAINING_DATA_URL")
```

Comment:
This is a big question. Some explanation algorithms need (or work better) if you have the training data. So I suppose my rationale is if it can be specified in the resource spec then probably should be an argument. Its probably important enough to be there so will add as optional argument with validation based on method.